src/lib.rs

```diff
@@ -92,11 +92,57 @@ impl<P: Pipe> PythonParserBuilder<P> {
 
     pub fn load_and_init_class<'p>(py: Python<'p>, module_name: &str, class_name: &str, options: &[(String, String)]) -> PyResult<PyObject> {
         let module = try!(Self::load_module(py, module_name));
+        let dict = module.dict(py);
+
+        if dict.contains(py, "error").unwrap() {
```

Comment:
try! will return an error, while unwrap() aborts the thread. It's ok to call unwrap() in test code, but it's usually avoided in production code.